### PR TITLE
proposed 2.16.3 changelog contents

### PR DESCRIPTION
**Pull request by mikehardy**
If approved, this will be the changelog baked into the 2.16.3 release

### CODE REVIEW

File: changelog.asc [hunk 1]
```diff
@@ -2,6 +2,21 @@
...
+* https://opencollective.com/ankidroid[🤜🤛 Thank you! Your support makes the fixes happen!]
+* Fix inconsistent italics between Anki / AnkiDroid
+* Fix crash in certain cases adding images
+* Fix display of saved card searches
+* Add ability to map "reschedule" to a gesture
+* Fix crash exporting decks with '/' in deck name
+* Fix crash displaying whiteboard menu
+* Allow long preference titles to wrap
+* Allow 3rd party cookies in webview to match Anki behavior
+* Improve "add shortcut" icon
```
Comment (BrayanDSO) [inline changelog.asc:17]:
```suggestion
* Improve "add" shortcut icon
```

File: changelog.asc [hunk 2]
```diff
@@ -2,6 +2,21 @@
...
+* Sync community-provided translations from https://crowdin.com/project/ankidroid
```
Comment (BrayanDSO) [inline changelog.asc:18]: nit: don't follow if you don't want, but some people may not understand what `Sync` would mean here
```suggestion
* New community-provided translations from https://crowdin.com/project/ankidroid
```

**Comment by snowtimeglass** [comment]
Won't the merge of [#14234](https://github.com/ankidroid/Anki-Android/pull/14234) be ready in time for release of 2.16.3?

I have the same impression as dae does. 

>There have been quite a few reports of this, so it might be worth addressing in a 2.16.3 before switching over to a 2.17 branch.

[#14225](https://github.com/ankidroid/Anki-Android/issues/14225)

**Comment by lukstbit** [comment]
The changelog should mention the removal of the Beolingus feature in  [#14181](https://github.com/ankidroid/Anki-Android/pull/14181)

File: changelog.asc (hunk 1 above)
Comment (mikehardy) [reply changelog.asc]: I removed the quotes entirely based on this suggestion

File: changelog.asc (hunk 2 above)
Comment (mikehardy) [reply changelog.asc]: I like that better, thank you

**Comment by mikehardy** [comment]
- added content based on my sweep through the PR queue just now
- and I fixed the date in the header
- organized into general structure of "announce+thanks" / "crash fixes" / "general fixes" / "other changes" as I think crashes are usually the highlight
 
re-pushed

**Pull request merged**